lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  final sections = ['People', 'Language', 'System', 'Help'];
+  final List<Map<String, List<Object>>> features = <Map<String, List<Object>>>[
+    {
+      'Guest': [Icons.account_circle]
+    },
+    {
+      'Language': [Icons.language, '/speech'],
+      'Speech': [Icons.record_voice_over, '/speech']
+    },
+    {
+      'Export': [Icons.cloud_upload, '/speech'],
+      'Import': [Icons.cloud_download, '/speech'],
+      'Display': [Icons.remove_red_eye, '/display'],
+      'Scanning': [Icons.center_focus_strong, '/speech'],
+      'Navigation': [Icons.chevron_right, '/navigation']
+    },
+    {
+      'User Help': [Icons.help, '/user help'],
+      'About Cboard': [Icons.info, '/about'],
+      'Donate': [Icons.monetization_on, '/speech'],
+      'Feedback': [Icons.feedback, '/speech']
+    },
+  ];
+  @override
+  Widget build(BuildContext context) {
+    return ListView.separated(
+      separatorBuilder: (BuildContext context, int index) {
+        return SizedBox(
+          height: 10,
+        );
+      },
+      itemCount: sections.length,
+      itemBuilder: (BuildContext context, int index) {
+        return Container(
+          padding: EdgeInsets.all(16.0),
+          decoration: new BoxDecoration(
+              color: Colors.white, borderRadius: BorderRadius.circular(5.0)),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                sections[index],
+                textAlign: TextAlign.left,
+              ),
+              ListView.separated(
+                  separatorBuilder: (BuildContext context, int index) =>
+                      Divider(color: Color(0xffE0E0E0), indent: 78),
+                  physics: const NeverScrollableScrollPhysics(),
+                  shrinkWrap: true,
+                  itemCount: features[index].length,
+                  itemBuilder: (BuildContext context, int iconIndex) {
+                    return GestureDetector(
+                      onTap: index != 0
```

Comment:
Here, there are two instances where you've checked the `index` of a list item. Do you think we should instead just create a separate `People` setting to avoid this logic here?

Reply:
good suggestion. Fixed it!

Reply:
perfect! Just did it. Please check again if it makes sense

Reply:
I'm not sure if I see what you did. I was mentioning that instead of checking if `index == 0`, we create a separate `People` component and remove that from this section entirely. This'll help us prevent from having to check the index so many times - does this make sense?

Reply:
I put the `People` section separately already and we just check index once.

Reply:
If the `People` section is separated, why do we still need to check the index? Sorry, I might be missing it 😅 !

Reply:
I used ListView to display the sections so I'd need to include `People` there. But since only the `LOGIN/SIGN UP` button is different, I check index once for that.